packages/api/src/external/ehr/elation/command/process-patients-from-appointments.ts

```
@@ -39,6 +42,17 @@ export async function processPatientsFromAppointments(): Promise<void> {
     return;
   }
 
+  const secondaryMappingsMap = cxMappings.reduce((acc, cxMapping) => {
+    if (!cxMapping.secondaryMappings) {
+      throw new MetriportError("Elation secondary mappings not found", undefined, {
+        externalId: cxMapping.externalId,
+        source: EhrSources.elation,
+      });
+    }
+    const secondaryMappings = elationSecondaryMappingsSchema.parse(cxMapping.secondaryMappings);
+    return { ...acc, [cxMapping.externalId]: secondaryMappings };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)